docs/advanced/api_examples.yml

```diff
@@ -270,14 +270,16 @@
 
```

Comment:
I suggest adding an example per feature, given that the ones you're adding now are optional parameters.